packages/nextjs/app/builders/0x5defa248834d4b6112a0Ad3854AA22913e01BD83/page.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import Image from "next/image";
+import type { NextPage } from "next";
+
+const MichaelNwachukwuProfile: NextPage = () => {
+  return (
+    <main className="flex max-h-screen justify-center items-center h-full w-full pt-16">
+      <div className="flex flex-col items-center gap-2 max-w-xl">
+        <Image
+          src="/michael-nwachukwu.png"
```

Comment:
As more builder profiles are added, you might want to consider creating a dedicated folder for your images within the public folder.